ui/raidboss/raidboss_config.ts

```diff
@@ -684,11 +688,24 @@ class RaidbossConfigurator {
         if (!hasOutputFunc && !this.base.developerOptions)
           continue;
 
+        const triggerDivId = document.createElement('div');
+        triggerDivId.classList.add('trigger');
+
         // Build the trigger label.
         const triggerDiv = document.createElement('div');
-        triggerDiv.innerHTML = trig.isMissingId ? '(???)' : trigId;
+        triggerDivId.innerHTML = trig.isMissingId ? '(???)' : trigId;
+        triggerDivId.classList.add('trigger-id');
+        triggerDiv.appendChild(triggerDivId);
+
+        // Build the trigger explanatory label
+        const trigLabel = trig.label?.[this.displayLanguage] ?? trig.label?.en;
+        if (trigLabel && trigLabel !== triggerDiv.innerHTML) {
+          const triggerLabel = document.createElement('div');
+          triggerLabel.innerHTML = trigLabel;
+          triggerLabel.classList.add('trigger-label');
```

Comment:
I think you should use a different CSS class here, as `trigger-label` is also used in the `addTriggerDetail` function on line 478 for the headings for each of the trigger fields. You could also call this "trigger comment" instead of "label" if you needed a different word.

Once you add a different CSS class, my only other suggestions are:

(1) I think it would be nice to have a little bit of vertical whitespace between the trigger id and the trigger label
(2) It would be nice to handle very long text better; right now very long text just pushes the right part of the grid over and it's very hard to read

Something like this will help the label be more visually distinctive and not look like part of the id:

```css
/* there's already a .trigger-label, so need something else here */
.trigger-comment {
  display: block; /* this might not be needed */
  font-size: 12px;
  margin-top: 8px;
  margin-left: 8px;
}
```

Something like this will help very long text (maybe 300px or 400px or something like that?)

```diff
 .trigger-details {
   display: grid;
-  grid-template-columns: repeat(1, auto 1fr);
+  grid-template-columns: repeat(1, minmax(auto, 300px) 1fr);
   grid-gap: 3px 10px;
 }
```